test/integration/redis-cert/test.sh

```diff
@@ -39,3 +45,12 @@ redis_cert_test() {
     fi
     rm output.log
 }
+
+wait_for_redis() {
+    timeout 30 bash -c '
+    while true; do
+        kubectl logs -n connaisseur -lapp.kubernetes.io/instance=redis --tail=-1 | grep -q  "Ready to accept connections tls" && break
+        sleep 1
+    done
+    '
```

Comment:
You could fail open here and hope it still works
```suggestion
wait_for_redis() {
    timeout 30 bash -c '
    while true; do
        kubectl logs -n connaisseur -lapp.kubernetes.io/instance=redis --tail=-1 | grep -q  "Ready to accept connections tls" && break
        sleep 1
    done
    ' || true
```
That would allow not breaking the test if the message changes, but also prolong the test by 30s everytime if it changed. Your call (I guess I wouldnt do it, but you're more vocal on test failures; I just had the thought that one could do it)